component/storage/indexeddb/indexeddb.go

```diff
@@ -96,14 +99,26 @@ func (p *Provider) OpenStore(name string) (storage.Store, error) {
 		return nil, err
 	}
 
+	newlyCreatedStore := p.stores[name]
+
+	// Create the tag map if it doesn't exist already.
+	_, err = p.stores[name].Get(tagMapKey)
+	if errors.Is(err, storage.ErrDataNotFound) {
+		err = newlyCreatedStore.Put(tagMapKey, []byte("{}"))
```

Comment:
how about empty bytes??

Reply:
It has to be unmarshallable to a `tagMapping`. Empty bytes cause a failure, while `{}` allows it to marshal to an empty `tagMapping` (which is desired).